OptionSelector/CQDetailTableViewController.m

```diff
@@ -0,0 +1,119 @@
+//
+//  CQDetailTableViewController.m
+//  Unit1WeekFourHomework
+//
+//  Created by Jackie Meggesto on 8/12/15.
+//  Copyright (c) 2015 Jackie Meggesto. All rights reserved.
+//
+
+#import "CQDetailTableViewController.h"
+
+@interface CQDetailTableViewController ()
+@property (nonatomic) NSIndexPath *checkedIndexPath;
+
+
+@end
+
+@implementation CQDetailTableViewController
+
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    
+    
+    
+    
+    // Uncomment the following line to preserve selection between presentations.
+    // self.clearsSelectionOnViewWillAppear = NO;
+    
+    // Uncomment the following line to display an Edit button in the navigation bar for this view controller.
+    // self.navigationItem.rightBarButtonItem = self.editButtonItem;
+}
+
+-(void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath *)indexPath {
+    self.checkedIndexPath = indexPath;
+    self.category.cqIndexPath = indexPath;
```

Comment:
Storing the index path as part of the data model is redundant with storing the `selection` (the line below). For a given index path, you should be able to tell if that index path is the one for the selected option just by looking at the `selection` property on your model.